mmedit/models/editors/nafnet/baseline_net.py

```diff
@@ -0,0 +1,287 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from mmengine.model import BaseModule
+
+from mmedit.registry import MODELS
+from .local_utils import Local_Base
+from .utils import LayerNorm2d
+
+
+@MODELS.register_module()
+class Baseline(BaseModule):
+    """The original version of Baseline model in "Simple Baseline for Image
+    Restoration".
+
+    Args:
+        img_channels (int): Channel number of inputs.
+        mid_channels (int): Channel number of intermediate features.
+        middle_blk_num (int): Number of middle blocks.
+        enc_blk_nums (List of int): Number of blocks for each encoder.
+        dec_blk_nums (List of int): Number of blocks for each decoder.
+    """
+
+    def __init__(self,
+                 img_channel=3,
+                 mid_channels=16,
+                 middle_blk_num=1,
+                 enc_blk_nums=[],
```

Comment:
please check the default value of enc_blk_nums

Reply:
Done.